### PR TITLE
Application: Remove handler for impossible case

### PR DESCRIPTION
**Pull request by danrabbit**
Because of these lines:
```
if (previous_plugs.size > 1) {
    navigation_button.label = previous_plugs.@get (0).display_name;
    previous_plugs.remove_at (previous_plugs.size - 1);
} else {
    navigation_button.label = _(all_settings_label);
}
```

It's impossible for the back button to be anything other than `all_settings_label`  and `previous_plugs` to be smaller than 1. So we don't need to handle a case that isn't possible to get into.

This is also nice because now it means `switch_plug` is only called by `load_plug` and we can combine these

### CODE REVIEW

File: src/Application.vala [hunk 1]
```diff
@@ -413,12 +421,8 @@ namespace Switchboard {
...
                 deck.visible_child = category_view;
             } else {
-                if (previous_plugs.size > 0) {
-                    load_plug (previous_plugs.@get (0));
-                    previous_plugs.remove_at (0);
-                } else {
-                    switch_to_plug (current_plug);
-                }
+                load_plug (previous_plugs.@get (0));
+                previous_plugs.remove_at (0);
```
Comment (jeremypw) [inline src/Application.vala:425]: `previous_plugs` is removed by PR #196  so this will fail if that PR is merged.

**Review by jeremypw — CHANGES_REQUESTED** [review]
Conflicts with #196

File: src/Application.vala (hunk 1 above)
Comment (danrabbit) [reply src/Application.vala:425]: That's `loaded_plugs` which is a separate list

Comment (jeremypw) [reply src/Application.vala:425]: Oops - ignore that.

**Review by jeremypw — APPROVED** [review]
Works as expected - no new issues found

**Pull request merged**